### PR TITLE
Import installed python modules.

### PR DESCRIPTION
**Pull request by oliverchang**
For #22.

### CODE REVIEW

File: build/python/analyze.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        print('Importing', import_path)
+        try:
+            importlib.import_module(import_path)
+        except:
+            print('Failed to import', import_path)
+            traceback.print_exc()
+
+
+def main():
+    package_with_version = sys.argv[1]
```
Comment (laurentsimon) [inline build/python/analyze.py:43]: maybe check length f sys.args and raise ValueError() if not equal to 1?

Comment (oliverchang) [reply build/python/analyze.py:46]: sure, added.

File: build/python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+FROM python:3
```
Comment (inferno-chromium) [inline build/python/Dockerfile:1]: Is folder name python better or pypi ?

**Review by inferno-chromium — APPROVED** [review]

File: analysis/python-runner.sh [hunk 3]
```diff
@@ -2,9 +2,9 @@
```
[the remark this answers is not in the document]
Comment (oliverchang) [reply analysis/python-runner.sh:2]: None of the existing files in this repo have license headers. Can add all together later.

File: build/python/Dockerfile (hunk 2 above)
Comment (oliverchang) [reply build/python/Dockerfile:1]: I think it's better to follow the upstream docker image naming here. Can easily change later.

File: analysis/python-runner.sh [hunk 4]
```diff
@@ -2,9 +2,9 @@
 
 for pkg in $(cat python.txt); do
   rand=$(cat /dev/urandom | tr -dc 'a-z0-9' | head -c 10)
-  kubectl run pypi-$rand --image=python:3 --restart='Never' \
+  kubectl run pypi-$rand --image=gcr.io/ossf-malware-analysis/python --restart='Never' \
```
[the remark this answers is not in the document]
Comment (oliverchang) [reply analysis/python-runner.sh:5]: Done!

**Review by laurentsimon — APPROVED** [review]

**Pull request merged**